src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
do not use margin and padding with `*`-selector, style margins and paddings of particular elements